### PR TITLE
Module7 ci cd

### PR DESCRIPTION
**Pull request by jainsushma**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+script:
+  # Build
+  - docker build --target test --tag todo-app:test .
+
+  #Tests
+  #Unit & Integration tests
+  - docker run todo-app:test tests
+
+  # End to End test
+  - docker run -e SECRET_KEY=$SECRET_KEY -e TOKEN=$TOKEN todo-app:test tests_e2e
```
Comment (JackMeadDev) [inline .travis.yml]: Really minor, but we can take advantage of a short-syntax here by switching to:
```suggestion
  - docker run -e SECRET_KEY -e TOKEN todo-app:test tests_e2e
```

File: README.md [hunk 2]
```diff
@@ -68,7 +68,18 @@ docker run --env-file .env -p 5000:5000 --mount type=bind,source="$(pwd)"/todo_a
 docker build --target development --tag todo-app:dev .
 
 Pass environment variables as a file, publish the application on port 5001 and listen on port 5001
-docker run --env-file .env -p 5001:5001 --mount type=bind,source="$(pwd)"/todo_app,target=/todo-app todo-app:dev
+docker run --env-file .env -p 5001:5001 --mount type=bind,source="$(pwd)"/todo_app,target=/todo-app todo-app:prod
+```
+
+## Running the tests in multi-stage docker file in test
```
Comment (JackMeadDev) [inline README.md:74]: Nice clear instructions!

**Review by JackMeadDev — APPROVED** [review]
Nice work, I can see that Travis has approved this PR as the tests are running and passing there, and I was able to easily run your tests in docker following your instructions. I've left a couple of minor comments below, but happy for you to merge this when you're ready

File: .travis.yml (hunk 1 above)
Comment (jainsushma) [reply .travis.yml]: Done

File: README.md (hunk 2 above)
Comment (jainsushma) [reply README.md:74]: Thank you :)

**Pull request merged**